index.js

```diff
@@ -1,2 +1,7 @@
-const numbers=[1,2,3,4,5];
-console.log(numbers);
\ No newline at end of file
+// Q1:Declare an empty array;
+let arr=Array(); 
```

Comment:
@Fakhara77 there are many ways to declare an empty array, can you please add all the ways?